test/Passes/xsmm-combine.mlir

```diff
@@ -0,0 +1,250 @@
+//RUN: tpp-opt --combine-xsmm-op-optimization %s --split-input-file | FileCheck %s
+
+// CHECK-LABEL: func.func @entry(
+// CHECK: %[[ARG0:.*]]: memref<256x128xf32>) -> memref<256x512xf32> {
+// CHECK: xsmm.fused_brgemm.dispatch
+// CHECK: xsmm.fused_brgemm
+
+memref.global "private" constant @__constant_4x32x32xf32 : memref<4x32x32xf32> = dense<1.000000e+00> {alignment = 128 : i64}
+memref.global "private" constant @__constant_8x32x32xf32 :  memref<8x32x32xf32> = dense<1.000000e+00> {alignment = 128 : i64}
+memref.global "private" constant @__constant_32xf32:  memref<32xf32, strided<[32], offset:?>> = dense<1.000000e+00> {alignment = 128 : i64}
+
+//bcast_col_in0 flag set on binary add 
```

Comment:
Instead of comment + generic function name, change the function name to what the comment says, then it's east to map what CHECK lines apply to what functionality, not just what split test.